resources/views/home/landing.blade.php

```diff
@@ -0,0 +1,203 @@
+                <a href="#" class="landing-nav__section__link" title="{{ trans("users.anonymous.login_link") }}" data-toggle="modal" data-target="#user-dropdown-modal">{{ trans("users.login._") }}</a>
+                <a href="{{ route("users.register") }}" class="landing-nav__section__link">{{ trans("users.signup._") }}</a>
+            </div>
+        </div>
+        <div id="user-dropdown-modal" class="modal fade" tabindex="-1">
+            <div class="modal-dialog modal__dialog js-user-dropdown-modal__dialog">
+                @if (Auth::check())
+                <div class="js-react--user-card"></div>
+                @else
+                <div class="modal-content modal-content--no-shadow">
+                    <div class="modal-header modal-header--login"><h1 class="modal-header__title">{{ trans("users.login._") }}</h1></div>
+                    <div class="modal-body modal-body--user-dropdown modal-body--no-rounding">
+                        <h2 class="modal-body__title modal-body__title">{{ trans("users.login.title") }}</h2>
+
+                        {!! Form::open(["url" => route("users.login"), "id" => "login-form", "class" => "modal-body__form form", "data-remote" => true]) !!}
+                        <div class="form__input-group form-group form-group--compact">
+                            <input class="modal-af form-group__control form-control form-group__control--compact" name="username" type="text" placeholder="{{ trans("users.login.username") }}" required>
+                            <input class="form-group__control form-control form-group__control--compact" name="password" type="password" placeholder="{{ trans("users.login.password") }}" required>
+                        </div>
+
+                        <button class="btn-osu btn-osu-default form__button" type="submit"><i class="fa fa-sign-in"></i></button>
+                        {!! Form::close() !!}
+
+                        <p class="modal-body__paragraph"><a href="{{ route("users.forgot-password") }}" target="_blank">{{ trans("users.login.forgot") }}</a></p>
+                        <p class="modal-body__paragraph"><a href="{{ route("users.register") }}" target="_blank">{{ trans("users.login.register") }}</a></p>
+                    </div>
+                </div>
+                @endif
+            </div>
+        </div>
+    </nav>
+    <div id="popup-container">
+        <div class="alert alert-dismissable popup-clone col-md-6 col-md-offset-3 text-center" style="display: none">
+            <button type="button" data-dismiss="alert" class="close"><i class="fa fa-close"></i></button>
+            <span class="popup-text"></span>
+        </div>
+    </div>
+    <header class="osu-layout__row osu-layout__row--landing">
+        <div class="landing-hero">
+            <div class="js-landing-hero-slider landing-hero-slider">
+                @for($i = 1; $i <= 2; $i++)
+                <a href="#" class="landing-slide">
```

Comment:
As currently there's nothing to link it to just make it a span/div/not a link for now. Proper implementation probably involves a table which contains images and urls.